docs/HowTo/Interact/APIs/Engine-API.md

```diff
@@ -0,0 +1,193 @@
+---
+description: How to enable and use the Engine API
+---
+
+# Engine API
+
+The Engine API allows communication between [consensus and execution layers](../../../Concepts/Merge.md#execution-and-consensus-clients) of the [post-Merge](../../../Concepts/Merge.md) Ethereum Client.
+These [API methods](../../../Reference/Engine-API-Methods.md) are a separate subsection of the [JSON-RPC API](API.md).
+
+## Configuring the Engine API
```

Comment:
```suggestion
## Configure the Engine API
```

Use simple present tense of verbs in headings.

Reply:
Fixed